spring/src/main/java/io/cloudevents/spring/messaging/CloudEventsHeaders.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2018-Present The CloudEvents Authors
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.cloudevents.spring.messaging;
+
+public class CloudEventsHeaders {
+
+	public static final String CE_PREFIX = "ce-";
```

Comment:
This prefix doesn't work for all transports, how do we deal with that?

Reply:
It will work for all transports because Spring will convert the transport specific headers to the canonical form. We control that on the Spring side - we can agree that is the contract and it will just work. My next job is to create some samples that demonstrate.